sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
When you say there was only one code change, which one are you talking about? Changing the package from main -> cmd? 
Can you cut & paste your sc-cli.go that creates a binary from the cmd directory? I'm having a hard time understanding / replicating a working binary from there :)

Just to get it to compile, I had to:
 - change package main->cmd
 - remove the import statement
 - cmd.RootCmd => RootCmd